src/pages/experimental/create-flows/index.mdx

```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
 
-When any create action is a success, a notification banner can be presented to visually confirm this.
+</Column>
+</Row>
 
-![Example of a successful creation confirmation"](images/16.png)
+<Row>
+<Column colLg={8}>
 
-<Caption>Example of a successful creation confirmation</Caption>
+![Button position in context when scrolling.](images/12.png)
 
-## Anatomy
+<Caption>Buttons should stick upon scrolling. </Caption>
 
-### Button position
+</Column>
+</Row>
 
-When viewed on mobile, the buttons are positioned at the bottom of the screen.
+When the full screen dialog is viewed in mobile, the buttons are positioned at the bottom of the screen.
 
 <Row>
 <Column colLg={8}>
 
-![Button position in context.](images/6.png)
+![Button position in context on mobile.](images/13.png)
 
-<Caption>Example of full-page option on mobile</Caption>
+<Caption>Example of full-page option on mobile.</Caption>
 
 </Column>
 </Row>
 
-On a larger screen, buttons should always remain visible to the user.
-
-If a display is likely to extend beyond the visible display, buttons should remain visible and should stick upon scrolling. If only a few buttons are present, they can be placed on the left below the last field.
-
 <Row>
 <Column colMd={4} colLg={6}>
-<DoDontExample caption="Buttons should not be hidden off-screen.">
+<DoDontExample type="do" caption="Buttons should stick upon scrolling so they are always visible.">
 
-![Buttons should not be hidden off-screen.](/images/7.png)
+![Buttons should stick upon scrolling so they are always visible.](/images/14a.png)
 
 </DoDontExample>
 </Column>
 <Column colMd={4} colLg={6}>
-<DoDontExample type="do" caption="Buttons are located on the left if they are likely to be visible when the page loads.">
+<DoDontExample type="do" caption="If an illustration or image is displayed then buttons should be displayed above the image.">
 
-![Buttons are located on the left if they are likely to be visible when the page loads.](/images/7.7.png)
+![If an illustration or image is displayed on the right then buttons should be displayed above the image.](/images/14b.png)
 
 </DoDontExample>
 </Column>
 </Row>
 
+### Including an illustration/image
+
+If including a supporting image or illustration, the recommended place is on the right hand side.
+
 <Row>
-<Column colMd={4} colLg={6}>
-<DoDontExample type="do" caption="Buttons located on the right should stick upon scrolling so they are always visible.">
+<Column colLg={8}>
 
-![White overflow menu on Gray 10 card.](/images/7.8.png)
+![Example of supporting image placement.](images/15.png)
+
+<Caption>Example of supporting image placement.</Caption>
 
-</DoDontExample>
 </Column>
+</Row>
+
+### Show all available options toggle
+
+The toggle should be used to avoid overwhelming users, whilst still allowing users the ability to access everything. It can be applied to the [side panel](#side-panel) and the [full screen](#full-screen) dialog.
```

Comment:
```suggestion
The toggle should be used to avoid overwhelming users, while still allowing users to access all options. It can be applied to the [side panel](#side-panel) and the [full screen](#full-screen) dialog.
```